src/util/dstring.h

```diff
@@ -13,6 +13,9 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include "string_container.h"
 
+class serializert;
+
```

Comment:
Your argument sounds like it could be used to say we only need one type of header in documents. Different things need different levels of separation. I'll implement this because I'm giving up trying to make nice code on this project but I wish I could have made things better here.